generators/server/templates/src/test/java/package/web/rest/_UserResourceIntTest.java

```diff
@@ -192,12 +195,36 @@ public static User createEntity(<% if (databaseType === 'sql') { %>EntityManager
         return user;
     }
 
+    <%_ if (databaseType === 'sql') { _%>
+    /**
+     * Search for the last inserted user id.
+     */
+    public static String getLastInsertedId(EntityManager em){
+        String lastInsertedIdAsString = "";
+        List<User> userList = em.createQuery("select u from User u").getResultList();
```

Comment:
We should use proper queries - even in tests : 
`Long maxId = em.createQuery("select max(u.id) from User u").getSingleResult()`

Reply:
10x for the feedback. I totally agree with you on the sorting part. On the second point, regarding the use of singleResult, I want to overcome e NoResultException if  no user exist in the database. I will commit the requested changes.